src/screens/conclusion/index.js

```diff
@@ -1,73 +1,85 @@
-import "./conclusion.css";
+import { PatreonIcon } from "../../components/patreon-icon";
 
 export function Conclusion() {
   return (
-    <section id="conclusion" className="together-we-can screen-content">
-      <div className="full-screen bears h-full w-full flex-col justify-between">
-        <div className="h-screen-50vh flex w-[100%] flex-col self-start py-16 pr-10 lg:w-[75%] 2xl:w-[50%]">
```

Comment:
@stephan281094 I simplified many of these width classNames and removed one of these divs because I don't think it was needed.  Unfortunately the diff got harder to read because of that.